eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/steps/processors/ContextNucleotideAdditionProcessor.java

```diff
@@ -36,20 +36,21 @@ public ContextNucleotideAdditionProcessor(FastaSynonymSequenceReader fastaReader
 
     @Override
     public IVariant process(Variant variant) throws Exception {
-        long oldStart = variant.getStart();
         String contig = variant.getChromosome();
-        String oldReference = variant.getReference();
-        String oldAlternate = variant.getAlternate();
 
         if (fastaSequenceReader.doesContigExist(contig)) {
-            return getVariantWithContextNucleotide(variant, contig, oldStart, oldReference, oldAlternate);
+            return getVariantWithContextNucleotide(variant);
         } else {
-            throw new IllegalArgumentException("Contig '" + contig + "' does not appear in the fasta file ");
+            throw new IllegalArgumentException("Contig '" + contig + "' does not appear in the FASTA file ");
         }
     }
 
-    private IVariant getVariantWithContextNucleotide(Variant variant, String contig, long oldStart,
-                                                     String oldReference, String oldAlternate) {
+    private IVariant getVariantWithContextNucleotide(Variant variant) {
+        long oldStart = variant.getStart();
+        String contig = variant.getChromosome();
+        String oldReference = variant.getReference();
+        String oldAlternate = variant.getAlternate();
+
         String newReference;
         String newAlternate;

```

Comment:
These 2 variables can now be declared inside the `if`, because they are not used otherwise.